contribute.py

```diff
@@ -92,6 +97,13 @@ def arguments(argsval):
                         from 1 to N times a day. The exact number of commits
                         is defined randomly for each day. The default value
                         is 10.""")
+    parser.add_argument('-mic', '--min_commits', type=int, default=1,
```

Comment:
Thank you for the contribution.
Considering there is already a `-mc` option, it is a bit confusing to have `-mic` and `-mc` at the same time. Could you please rename `-mc` to `-maxc` and `-mic` to `-minc` to make it more transparent?

Reply:
done